OWL2/OWL22CASL.hs

```diff
@@ -495,77 +496,77 @@ mapCard b cSig ct n prop d var = do
         exactLst' = mkImpl (conjunct $ oPropsE ++ fOut) $ disjunct dlstM
         senAux = conjunct [minLst, mkForall qVarsE exactLst']
         exactLst =  if null qVars then senAux else mkExist qVars senAux
-        ts = uniteL $ [cSig] ++ s ++ s' ++ s''
+        ts = concat $ s ++ s' ++ s''
```

Comment:
why can `csig` be omitted here (and below)?

Reply:
csig is not needed, it was passed around just so we can return a signature when no change happens, and to be extended with the new symbols. I will remove it throughout the code.

Reply:
Also, note that csig is added at the union in mapTheory anyways, and you don't want to have it there more than once.